Algorithm/QCAlgorithm.Python.cs

```diff
@@ -34,14 +34,14 @@ namespace QuantConnect.Algorithm
 {
     public partial class QCAlgorithm
     {
-        private PandasConverter _converter;
+        public PandasConverter PandasConverter { get; private set; }
 
         /// <summary>
         /// Sets pandas converter
         /// </summary>
-        public void SetPandas()
+        public void SetPandasConverter()
```

Comment:
No change required, but just a design comment. Typically a `SetXXX` method would accept the value to be set. In this case, something like `SetPandasConverter(IPandasConverter)` or something similar. Once again, not recommending the change now, just providing some general design feedback here.